### ISSUE TITLE
Screen flickering

### ISSUE THREAD

**Issue by winstonma** [latin-1/um5302ta#7]
Hi, I  would like to see if you have the same strange behavior

When I use my Linux device for 4-5 hours and I would saw my screen flickers. Would like to know if your laptop got the same behavior. The flickering looks very similar to [this youtube video](https://youtu.be/B2otGmQMqpA).

**Comment by latin-1** [comment]
Yes, I have the same problem under GNOME. However, some KDE users claim that they are not experiencing the same issue. I have no idea.

**Comment by winstonma** [comment]
Thanks for answering

I really need to check whether the behavior is expected or unexpected.

**Comment by winstonma** [comment]
Just an update.

It seems to me that the screen flickering issue is gone after installing the KDE and then [remove KDE completely](https://trendoceans.com/how-to-remove-the-kde-plasma-environment-in-ubuntu/). I am not sure what happened but I don't get screen flickering after since.

EDIT: Please ignore the above comment. I saw screen flicking.

**Comment by latin-1** [comment]
https://gitlab.freedesktop.org/drm/amd/-/issues/2352
You can also see the link above. It may or may not help. I'm sorry but I don't currently have time to test these workarounds.

**Comment by akoidan** [comment]
I have exactly the same issue, and it's starting to appear after some time as well. Sometimes I put my laptop to sleep (instead of poweroff), so I'm not sure whether it's connected, whether not, since I don't work straight for many hours.
